src/Symfony/Component/HttpClient/HttpClientTrait.php

```diff
@@ -181,6 +181,10 @@ private static function mergeDefaultOptions(array $options, array $defaultOption
                 }
             }
 
+            if ('auth_ntlm' === $name) {
+                throw new InvalidArgumentException(sprintf('Option "%s" is not supported by %s, try using CurlHttpClient instead.', __CLASS__));
```

Comment:
The option name is missing in the `sprintf()` call?

2 placeholders but only 1 value passed.

Reply:
good catch, could you please send a PR to fix it?

Reply:
Sure thing: https://github.com/symfony/symfony/pull/32476